### PR TITLE
Installation with oxide or draft

### PR DESCRIPTION
**Pull request by arnodeceuninck**
Based on a conversation on Discord.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,6 +132,17 @@ The app should start, play with it, but press **CTRL-C** (Hold the Control key a
 patches are cumulative (the last one contains all previous changes and gets updated with bugfixes)
 a patch can be applied more than once, it's more of a snapshot really, you can go back to a previous version
 
+## Installation with oxide or draft
+The installation starts an instance of xochitl, but isn't aware of oxide or draft running, 
+which needs xochitl to be started in a different way. 
+Execute this after the installation if you've got oxide or draft running:
```
Comment (Eeems) [inline README.md:138]: Execute this before the installation if you've got oxide or draft running:
```shell
systemctl stop tarnish # only if you're using oxide
systemctl stop draft # only if you're using draft
```

File: README.md [hunk 2]
```diff
@@ -132,6 +132,21 @@ The app should start, play with it, but press **CTRL-C** (Hold the Control key a
...
+The installation starts an instance of xochitl, but isn't aware of oxide or draft running, 
+which needs xochitl to be started in a different way. 
+Execute this before the installation if you've got oxide or draft running:
+```shell
+systemctl stop tarnish # only if you're using oxide
+systemctl stop draft # only if you're using draft
+```
+And this after the installation:
+```shell
+systemctl start tarnish # only if you're using oxide
```
Comment (Eeems) [inline README.md:145]:
```suggestion
systemctl stop xochitl
systemctl start tarnish # only if you're using oxide
```

Comment (Eeems) [reply README.md:145]: You still do need to make sure that xochitl isn't running before running tarnish or draft.

**Comment by Eeems** [comment]
This might need some tweaks to take into account rm2fb on the rM2. But I don't have one so I can't test.
@LinusCDE ?

**Comment by LinusCDE** [comment]
> This might need some tweaks to take into account rm2fb on the rM2. But I don't have one so I can't test.
> @LinusCDE ?

I think you just need to make sure that rm2fb is installed (which in turn enables rm2fb.service). If that is installed it seems to just work.

**Comment by arnodeceuninck** [comment]
> I think you just need to make sure that rm2fb is installed (which in turn enables rm2fb.service). If that is installed it seems to just work.

But that's not relevant for this readme, but rather for the readme of Oxide/Draft, since they won't work anyway without rm2fb installed

**Comment by LinusCDE** [comment]
> But that's not relevant for this readme, but rather for the readme of Oxide/Draft, since they won't work anyway without rm2fb installed

In that case I think you're trying to revert any launcher installations here.

On the rM 2 you need to **completely uninstall** rm2fb and ensure only xochitl will start on boot and not any other launcher. Just disabling `rm2fb.service` is not enough apparently and xochitl won't launch.

So adding a `opkg remove rm2fb` before disabling all other launchers and enabling xochitl should do the trick.

**Comment by Eeems** [comment]
> > But that's not relevant for this readme, but rather for the readme of Oxide/Draft, since they won't work anyway without rm2fb installed
> 
> In that case I think you're trying to revert any launcher installations here.
> 
> On the rM 2 you need to **completely uninstall** rm2fb and ensure only xochitl will start on boot and not any other launcher. Just disabling `rm2fb.service` is not enough apparently and xochitl won't launch.
> 
> So adding a `opkg remove rm2fb` before disabling all other launchers and enabling xochitl should do the trick.

I don't think the instructions are trying to remove the launchers, just disable them long enough to install remarkable-hacks.

**Comment by LinusCDE** [comment]
Ah ok. Got it wrong then. I think this shouldn't differ on the rM 2 then.

**Pull request merged**